packages/gatsby/src/redux/reducers/node-manifest.ts

```diff
@@ -12,7 +12,36 @@ export const nodeManifestReducer = (
 ): IGatsbyState["nodeManifests"] => {
   switch (action.type) {
     case `CREATE_NODE_MANIFEST`: {
-      const { manifestId, pluginName, node } = action.payload
+      const {
+        manifestId,
+        pluginName,
+        node,
+        updatedAt,
+        daysSinceLastUpdate = 30,
```

Comment:
I think this part of the api could be confusing - a user might think they should enter the number of days since the node was updated here. Maybe something like`maxDaysOld` 🤔

Reply:
I spent so long trying to think of a good variable name, and I knew this one wasn't good so I'm glad you came up with `maxDaysOld` 😆